### ISSUE TITLE
User 1326 displaying incorrectly in main members screen

### ISSUE THREAD

**Issue by pamelatech** [wid-tech/website#71]
**Describe the bug**
There are serious inconsistencies for a specific user.  Joanne Mavridis (user #1326) is an Admin if I go look at her membership in Apex, but she shows up in the apex members page as having the "Apply for Membership" plan.   If I look in her home chapter, she correctly shows up as a community member in her profile and the members page.

Database inconsistencies are concerning - @mcresilkstart  do you have any idea why this would happen?  I don't know how we could identify or fix these kinds of errors,  it may be we can fix by modulating the plan but it is still very concerning

**What kind of user are you?**
Site Admin

**To Reproduce**
1. Go to https://www.womeninidentity.org/onepage/profiles/joanne-mavridis/sections/membership
2. Note she is an admin
3. HEad to https://www.womeninidentity.org/onepage/members
4.Plan shows up as Apply for Membership

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/2591320/134811769-54c29269-a245-469f-a2b8-2793619396d9.png)

![image](https://user-images.githubusercontent.com/2591320/134811777-c4884b38-041e-409a-a53d-5ece51213ed4.png)
![image](https://user-images.githubusercontent.com/2591320/134811796-c1650641-8861-428e-b791-e8e023013f12.png)

**Comment by mcresilkstart** [comment]
@pamelatech that looks like a bug.  If you look at membership history on details for her you see she was initially on the 'Apply for Membership' plan and then was moved to the community plan.  The APEX members display is not correct for some reason.  I will report this as a bug. Note that being an Admin is more of a user status than a plan.  An admin typically exists with some kind of membership plan but doesn't have to have one.

**Comment by mcresilkstart** [comment]
@pamelatech I have logged this as a bug.  However I note that this behaviour is not consistent.  If you look at my test user Michael Creswell Test2 member number 1362 which started as an Apply for Membership and then got moved to Community member, this user is displayed correctly on the member list page.  This user was added at a later date than your example.  The problem that you found may be a hangover from something that is now fixed (bug fix of config fix).
